datasets/ua/war_sanctions/crawler.py

```diff
@@ -509,9 +552,16 @@ def crawl_legal_entity(context: Context, link, program):
         name = data.pop("Full name of legal entity")
     name_abbr = data.pop("Abbreviated name of the legal entity", None)
     reg_num = data.pop("Registration number")
+    rostec_ownership = data.pop("Within the structure of Rostec", None)
 
     legal_entity = context.make("LegalEntity")
-    legal_entity.id = context.make_id(name, name_abbr, reg_num)
+    # For companies owned by Rostec, we want to create simplified IDs
+    # to tie them to the parent company.
+    legal_entity.id = (
+        context.make_id(name_abbr, "affiliated with Rostec")
+        if rostec_ownership
+        else context.make_id(name, name_abbr, reg_num)
+    )
```

Comment:
since this is breaking onto multiple lines, perhaps we can allow ourselves a fully-fledged `if...else` as opposed to a ternary - that would be easier to read.